File: Source/OregonTilth.Web/src/app/shared/models/forms/harvest-post-harvest-standard-times/harvest-post-harvest-standard-time-summary-dto.ts
import { CropDto } from "../../generated/crop-dto";
import { CropUnitDto } from "../../generated/crop-unit-dto";
import { FieldLaborActivityDto } from "../../generated/field-labor-activity-dto";
import { FieldUnitTypeDto } from "../../generated/field-unit-type-dto";
import { HarvestTypeDto } from "../../generated/harvest-type-dto";
import { LaborTypeDto } from "../../generated/labor-type-dto";
import { MachineryDto } from "../../generated/machinery-dto";

export class HarvestPostHarvestStandardTimeSummaryDto {
    HarvestPostHarvestStandardTimeID: number;
    WorkbookID: number;
    CropID: number;
    Crop: CropDto;
    CropUnitID: number;
    CropUnit: CropUnitDto;
    HarvestTypeID: number;
    HarvestType: HarvestTypeDto;
    StandardTimePerUnit: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/HarvestPostHarvestStandardTime/HarvestPostHarvestStandardTimeSummaryDto.cs


using System.Collections.Generic;

namespace OregonTilth.Models.DataTransferObjects
{
    public class HarvestPostHarvestStandardTimeSummaryDto
    {
        public int? HarvestPostHarvestStandardTimeID { get; set; }
        public int WorkbookID { get; set; }
        public CropSummaryDto Crop { get; set; }
        public CropUnitDto CropUnit { get; set; }
        public HarvestTypeDto HarvestType { get; set; }
        public IEnumerable<TimeStudySummaryDto> TimeStudies { get; set; }
        
        
        
    }
}